src/ewatercycle/analysis/__init__.py

```diff
@@ -18,6 +33,7 @@ def hydrograph(
     precipitation_units: str = "mm day$^{-1}$",
     figsize: Tuple[float, float] = (10, 10),
     filename: Union[os.PathLike, str] = None,
+    nbars: int = 100,
```

Comment:
Why is the default 100? If I do not set nbar argument I expected it to either 
1. not do any aggregation or 
2. do aggregation so each bar is 1 pixel wide (figsize[0] / dpi).